src/vsr/replica.zig

```diff
@@ -9936,7 +9940,7 @@ pub fn ReplicaType(
                 return;
             }
 
-            if (self.journal.writing(message.header.op, message.header.checksum)) {
+            if (self.journal.writing(message.header) != .none) {
```

Comment:
I'd `switch` here, to make the logic more obvious at the call-site. Maybe even adjust the log message to be different?